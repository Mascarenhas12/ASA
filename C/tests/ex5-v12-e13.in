12
13
1 2
2 3
3 4
4 5
4 6
5 6
5 7
6 7
7 8
6 8
6 9
10 11
10 12