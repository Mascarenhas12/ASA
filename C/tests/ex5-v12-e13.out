2
9 12
5
3